packages/api/resolvers/mutations/createProductBundleItem.js

```diff
@@ -25,14 +25,5 @@ export default function createProductBundleItem(root, { productId, item }) {
   if (!bundleItem)
     throw new ProductNotFoundError({ productId: item.productId });
 
-  Products.update(productId, {
-    $set: {
-      updated: new Date(),
-    },
-    $push: {
-      bundleItems: item,
-    },
-  });
-
-  return Products.findOne(productId);
+  return product.createBundleItem({ item });
```

Comment:
I think this is more accurate like "addBundleItem"

Reply:
I mean the mutation is fine with "create" when you provide the item id and the product id but as a scoped function like "product.createBundleItem" it's a bit weird and it propably is better to have it more documenting what happens like:

"bundle.addItem()" or "product.addBundleItem()"...